modules/core/client/controllers/sendResetToken.controller.client.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+angular.module('nobsjs')
+  .controller('SendResetTokenController', SendResetTokenController);
+
+SendResetTokenController.$inject = ['$location', '$window', 'authService'];
+
+/**
+ * Manages the login and signup views
+ *
+ * @param {AngularService} $location Angular service that references the URL location in the browser
+ * @param {AngularService} $window Angular service that references the browser window
+ * @param {CustomService} authService that manages authentication
+ */
+function SendResetTokenController ($location, $window, authService) {
+  var vm = this;
+  vm.resetPassword = resetPassword;
+  vm.sendResetToken = sendResetToken;
+  vm.user = {};
+
+  //////////
+
+  function resetPassword () {
+    authService.resetPassword(vm.user)
+      .then(sendToLogin);
+  }
+
+  function sendResetToken () {
+    authService.sendResetToken(vm.user)
+      .then(sendToReset);
+  }
+
+  function sendToLogin () {
+    $location.path('/login');
```

Comment:
I believe best practice would be to use $state.go() vs $location.path() as we're using UI Router throughout the app